core/templates/dev/head/pages/exploration-player-page/learner-experience/tutor-card.directive.ts

```diff
@@ -187,7 +187,7 @@ oppia.directive('tutorCard', [
 
           $scope.toggleShowPreviousResponses = function() {
             $scope.arePreviousResponsesShown =
-             !$scope.arePreviousResponsesShown;
+            !$scope.arePreviousResponsesShown;
```

Comment:
This should be indented by 2. Put a '(' ')' around it if needed.

Reply:
Done!